tedium.ts

```diff
@@ -113,6 +116,13 @@ Generate a token here:   https://github.com/settings/tokens
   process.exit(1);
 }
 
+interface Config {
+  passes?: CleanupConfig;
+}
+let config: Config =
+    JSON.parse(stripJsonComments(fs.readFileSync('config.json', 'utf8')));
```

Comment:
I'd like to find another way to accomplish comments for the config file, but I'm fine with this for now, just making a note.